docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -91,9 +91,9 @@ For any third-party Cron scheduling tools, you can disable Crontab and add the f
 Disabling Crontab creates alerts after the universe is created, but they can be ignored. You need to ensure Cron jobs are set appropriately for YBA to function as expected.
   {{< /tip >}}
 
-- Verify that Python 3.6 or later is installed.
+- Verify that Python 3.5-3.8 is installed. v3.6 is recommended.
```

Comment:
Add step to set runtime config max version to whatever version of python > 3.7 that they are using.